app/controllers/shelters_controller.rb

```diff
@@ -22,4 +22,26 @@ def create
         shelter.save
         redirect_to "/shelters"
     end
+
+    def edit
+        @shelter = Shelter.find(params[:id])
+    end
```

Comment:
This is used to get the shelter in question to build the form in order to edit the shelter.